tests/infra/remote.py

```diff
@@ -428,6 +439,15 @@ def wait_for_stdout_line(self, line, timeout):
             "{} not found in stdout after {} seconds".format(line, timeout)
         )
 
+    def check_for_stdout_line(self, line, timeout):
+        for _ in range(timeout):
+            with open(self.out, "rb") as out:
+                for out_line in out:
+                    if line.strip() in out_line.strip().decode():
```

Comment:
Why `.strip()`?

Reply:
I am guessing to remove different whitespaces from around the actual string we are trying to find in the line?

Reply:
I think we should remove these `strip`s, so the search is the same as the grep version. On the off-chance someone adds whitespace into the `line` argument, they probably want it retained